Meetings/devel-call-2024-09-11.md

```diff
@@ -0,0 +1,40 @@
+# Meeting Minutes: SVSM Development Call (September 11th, 2024)
+
+## Attendees:
+
+Adam Dunlap, Diego Gonzalez Villalobos, Dionna Glaze, Elena Reshetova, Huibo Wang, James Bottomley, Joerg Roedel, Jon Lange, Nicolai Stange, Oliver Steffen, Peter Fang, Roy Hopkins, Stefano Garzarella, Tom Dohrmann, Tom Lendacky, Vijay Dhanraj
+
+## Topics:
+
+### COCONUT-SVSM BoF at LPC
+* The COCONUT-SVSM BoF was accepted for the next week's Linux Plumbers Conference.
+* It is scheduled on Thursday at 5 PM.
+* An email will be sent to the SVSM and KVM lists to invite people to the session.
+
+### Shadow Stacks
+* Tom Dohrmann is working on Shadow Stacks for SVSM and hopes to get a PR open this week.
+* He is troubleshooting one last issue and then it should be reviewable.
+* Joerg Roedel is curious to learn more about Shadow Stacks and review the PR.
+
+### Early Attestation and Measurement Architecture Document
+* Stefano Garzarella added a section to the document related to the rollback attack and clone attack.
+  * Not completed yet, maybe discuss in the BoF.
+* He also mentioned that Dionna updated the document with more details.
+
+### System Calls
+* Peter Fang gave an update on the system calls PR.
+* They are working on getting a PR ready for the initial stuff.
+* The PR will not include everything that they've done so far.
+
+### Page Strap Implementation
```

Comment:
Page Strap?

Reply:
Ah, leftover from the transcription, should have fixed that, thanks

Reply:
Fixed now.